### PR TITLE
FLUT-6235 - [Feature] UG document for DataGrid

### PR DESCRIPTION
**Pull request by Tamilarasan-Paranthaman**
## Description ##

I have prepared content for the custom sort icon feature and current cell is in editing enhancement for volume 1 2022 release.

### CODE REVIEW

File: Flutter/datagrid/editing.md [hunk 1]
```diff
@@ -564,6 +564,90 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
+## How to check whether current cell is in editing mode
+
+You can check whether current cell is in editing or not from the `SfDataGrid.isCurrentCellInEditing`. If current cell is in edit mode it will return true else return false.
```
Comment (NeelakandanKannan) [inline Flutter/datagrid/editing.md:569]: You can check whether current cell is in editing or not by using  `DataGridController.isCurrentCellInEditing` property.

File: Flutter/datagrid/sorting.md [hunk 2]
```diff
@@ -473,6 +473,91 @@ Widget build(BuildContext context) {
 
 ![flutter datagrid shows customized the sort icon color](images/sorting/flutter-datagrid-customized-sorticon-color.png)
 
+## Set custom sorting icon
+
+`SfDataGrid` allows to change the sort icon through [SfDataGridTheme.SfDataGridThemeData](https://pub.dev/documentation/syncfusion_flutter_core/latest/theme/SfDataGridThemeData-class.html) property. The DataGrid should be wrapped inside the `SfDataGridTheme`.
```
Comment (NeelakandanKannan) [inline Flutter/datagrid/sorting.md:478]: `SfDataGrid` allows to change the sort icon through using `SfDataGridThemeData.sortIcon` property. The DataGrid should be wrapped inside the `SfDataGridTheme`.  The given icon will be shown with the size (16*16).

File: Flutter/datagrid/sorting.md [hunk 3]
```diff
@@ -473,6 +473,91 @@ Widget build(BuildContext context) {
...
+
+{% tabs %}
+{% highlight Dart %}
+
+import 'package:syncfusion_flutter_core/theme.dart';
+
+{% endhighlight %}
+{% endtabs %}
+
+The sort icon can be changed by using [SfDataGridThemeData.sortIcon]. The following code describes how to change sort icon color by using [SfDataGridTheme]
```
Comment (NeelakandanKannan) [inline Flutter/datagrid/sorting.md:490]: This line is not required.

**Review by NeelakandanKannan — CHANGES_REQUESTED** [review]
Hi @Tamilarasan-Paranthaman ,

I have added my comments.

**Comment by Tamilarasan-Paranthaman** [comment]
rebuild

File: Flutter/datagrid/editing.md [hunk 4]
```diff
@@ -564,6 +564,90 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
+## How to check whether current cell is in editing mode
```
Comment (laksmipriyamurugesan) [inline Flutter/datagrid/editing.md:567]: How to check whether the current cell is in editing mode

File: Flutter/datagrid/editing.md [hunk 5]
```diff
@@ -564,6 +564,90 @@ Widget build(BuildContext context) {
...
+
+You can check whether current cell is in editing or not by using `DataGridController.isCurrentCellInEditing` property.
```
Comment (laksmipriyamurugesan) [inline Flutter/datagrid/editing.md:569]: You can check whether the current cell is in editing or not by using the `DataGridController.isCurrentCellInEditing` property.

File: Flutter/datagrid/sorting.md [hunk 6]
```diff
@@ -473,6 +473,89 @@ Widget build(BuildContext context) {
 
 ![flutter datagrid shows customized the sort icon color](images/sorting/flutter-datagrid-customized-sorticon-color.png)
 
+## Set custom sorting icon
```
Comment (laksmipriyamurugesan) [inline Flutter/datagrid/sorting.md:476]: Set a custom sorting icon

File: Flutter/datagrid/sorting.md [hunk 7]
```diff
@@ -473,6 +473,89 @@ Widget build(BuildContext context) {
...
+
+`SfDataGrid` allows to change the sort icon through using `SfDataGridThemeData.sortIcon` property. The DataGrid should be wrapped inside the `SfDataGridTheme`.
```
Comment (laksmipriyamurugesan) [inline Flutter/datagrid/sorting.md:478]: `SfDataGrid` allows you to change the sort icon by using the `SfDataGridThemeData.sortIcon` property.

File: Flutter/datagrid/sorting.md [hunk 8]
```diff
@@ -473,6 +473,89 @@ Widget build(BuildContext context) {
 
 ![flutter datagrid shows customized the sort icon color](images/sorting/flutter-datagrid-customized-sorticon-color.png)
 
+## Set custom sorting icon
+
+`SfDataGrid` allows to change the sort icon through using `SfDataGridThemeData.sortIcon` property. The DataGrid should be wrapped inside the `SfDataGridTheme`. 
+
+The `SfDataGridThemeData` and `SfDataGridTheme` classes are available in [syncfusion_flutter_core](https://pub.dev/packages/syncfusion_flutter_core) package. So, import the below file,
```
Comment (laksmipriyamurugesan) [inline Flutter/datagrid/sorting.md:480]: So, import the following file.

**Review by NeelakandanKannan — APPROVED** [review]
The changes are fine.

**Pull request merged**